driver/bpf/fillers.h

```diff
@@ -3518,6 +3518,12 @@ FILLER(sys_pagefault_e, false)
 	return res;
 }
 
+#ifdef SEND_SIG_FORCED
+#define SIGINFO_NOT_A_POINTER(_info) ((_info) <= SEND_SIG_FORCED)
```

Comment:
I don't think `<=` is defined behavior for pointers, and also we're relying on the implementation of `SEND_SIG_*`. We should explicitly check for each `SEND_SIG_*` type.

Reply:
In the patch that you helpfully sent me, look at is_sig_special. Looks like that's what the kernel itself uses (would be helpful if they just exported that dang function and saved us some trouble...).

If you'd rather me explicitly check each type though I can do so.

Reply:
Let's check explicitly. I would love to use the kernel implementation, and I doubt it will ever change, but there's nothing to guarantee that.